src/api/subscriptions.js

```diff
@@ -257,6 +257,16 @@ export const WindowOpenMode = {
   REDIRECT: 'redirect',
 };
 
+/**
+ * @enum {string}
+ */
+export const ReplaceSkuProrationMode = {
+  // The replacement takes effect immediately, and the remaining time will
+  // be prorated and credited to the user. This is the current default
+  // behavior.
+  IMMEDIATE_WITH_TIME_PRORATION: '1',
```

Comment:
The values should be strings as well - if we want to map them to numbers - we should do it internally. Realistically API users cannot use enums - they are here for our compilation needs and documentation. And we don't necessary want them to be passing us cryptic numbers.

Reply:
done